front50-gcs/src/main/java/com/netflix/spinnaker/front50/model/ApplicationBucketDAO.java

```diff
@@ -25,13 +25,11 @@
 import java.util.Collection;
 import java.util.Map;
 
-public class ApplicationBucketDAO extends BucketDAO<Application> implements ApplicationDAO {
-   public ApplicationBucketDAO(String basePath,
-                               StorageService service,
+public class ApplicationBucketDAO extends StorageServiceSupport<Application> implements ApplicationDAO {
```

Comment:
These DAOs will no longer exist after this refactor (rather we'll have *one* of each type backed by a `StorageService`.